test/e2e/kubectl.go

```diff
@@ -238,6 +240,48 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		var nsFlag string
+		var rcName string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
```

Comment:
This is just declaring strings, it doesn't need to be in a `BeforeEach` block. Just declare/init above.

[`Before|AfterEach()`](http://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach) is for sharing common setup/teardown of more "interesting" test state while keeping test env hermetic. For our purposes the shared state is pretty much just kubernetes resources. E.g. if you have a bunch of `It()` clauses that each spin up an rc and poke at it, you would put the rc creation in a `BeforeEach` so mutations by one test don't affect the others.